appintro-indicators/build.gradle

```diff
@@ -0,0 +1,30 @@
+apply plugin: 'com.android.library'
+apply plugin: 'kotlin-android'
+apply plugin: 'maven'
+apply plugin: 'signing'
+apply from: 'maven-push.gradle'
```

Comment:
You can probably drop this as well as the gradle script. Sonatype was used in the early versions of the library.